src/data/faq.json

```diff
@@ -282,8 +282,7 @@
                                 "anchor": "rat_multiple_rat",
                                 "active": false,
                                 "textblock": [
-                                    "No, only one rapid test can be registered in the app at a time. A negative rapid test is displayed in the app for 48 hours. After this time you can register a new test. You also have the option to remove the negative rapid test and result, and then register a new rapid test. Positive rapid tests are no longer displayed in the app once they have been shared.",
-                                    "PCR tests are handled independently from rapid tests. This means you can have one PCR test and one rapid test registered in the app at the same time."
+                                    "Yes, since version 2.21 it is possible to manage quick tests of family members in the Corona-Warn-App. For each person the following continues to apply: <ul><li>A negative rapid test is displayed in the app for 48 hours. After this time you can register a new test. You also have the option to remove the negative rapid test and result, and then register a new rapid test.</li><li>Positive rapid tests are no longer displayed in the app once they have been shared.</li><li>PCR tests are handled independently from rapid tests. This means you can have one PCR test and one rapid test registered in the app at the same time.</li></ul>"

```

Comment:
The last sentence is still no really clear, whether this applies to all tests in general or only tests from one person.

Reply:
See conversation above